src/routes/safe/components/Transactions/TxsTable/columns.tsx

```diff
@@ -30,14 +29,14 @@ const NOT_AVAILABLE = 'n/a'
 interface AmountData {
   decimals?: number | string
   symbol?: string
-  value: number | string
+  value: string
 }
 
 const getAmountWithSymbol = (
   { decimals = 0, symbol = NOT_AVAILABLE, value }: AmountData,
   formatted = false,
 ): string => {
-  const nonFormattedValue = new BigNumber(value).times(`1e-${decimals}`).toFixed()
+  const nonFormattedValue = humanReadableTokenAmount(value, decimals as number)
```

Comment:
nonFormattedValue and humanReadableTokenAmount names doesn't play well with each other